### ISSUE TITLE
GNOME shell integration

### ISSUE THREAD

**Issue by AlfioEmanueleFresta** [linux-credentials/libwebauthn#33]
Tracking reaching out to GNOME engs, and integration work.

I haven't reached out to anyone in the GNOME community yet. If you're reading this and you'd like to help, please reach out!

**Comment by ibotty** [comment]
This is the same as #7, right?

**Comment by AlfioEmanueleFresta** [comment]
@ibotty, good point, I had actually forgotten about #7. #7 only relates to UI/UX. I'd like to keep both open as someone could start working on the UI/UX independently.

**Comment by iinuwa** [comment]
Have you heard of Daiko Ueno's and Norbert Pócs's work on a similar goal?

https://blogs.gnome.org/dueno/bringing-fido2-device-support-to-sandboxes/
https://gitlab.freedesktop.org/dueno/fido2-proxy

**Comment by AlfioEmanueleFresta** [comment]
Closing this as this now belongs to @iinuwa's sibling project under [linux-credentials](https://github.com/linux-credentials).
